docs/contributing.md

```diff
@@ -1,16 +1,32 @@
 # Contributing
-Contributions are welcome. We have a few requests of any new contributors.
+Contributions are welcome. We have a few requests for new contributors:
 
-* Follow styling as designated in our [styling documentation](/docs/style.md).
-* Ensure that all changes which affect logic are covered by unit tests. 
-* Do not introduce any unit test failures/regressions.
-* Turn on automated github actions in your fork to have github run all the unit tests after pushing. See example below:
+* **Follow styling guidelines.**
+We have designated styling guideline to ensure uniformity and ease of communication. Please take a look at 
+the [styling documentation](/docs/style.md).
+
+* **Ensure that all changes are covered by unit tests.** 
+It is strongly recommended that unit tests are used to avoid regression and to ensure everything is still working.
+If you wish to contribute by adding a new game, please take a look at the [logic unit test documentation](/docs/world%20api.md#tests).
+If you wish to contribute to the website, please take a look at [these tests](/test/webhost).
+
+* **Do not introduce unit test failures/regressions.**
+Archipelago supports multiple versions of Python. You may need to download older Python versions to fully test
+your changes. Currently, the oldest supported version is [Python 3.8](https://www.python.org/downloads/release/python-380/). 
+It is recommended that automated github actions are turned on in your fork to have github run all of the unit tests after pushing.
+You can turn them on here:
 ![Github actions example](./img/github-actions-example.png)
 
-Otherwise, we tend to judge code on a case to case basis.
+Other than these requests, we tend to judge code on a case by case basis. For contribution to the website, please refer to
+[this document](/WebHostLib/README.md).
```

Comment:
The website part should probably be a bullet point above. Maybe merged with other sub-docs as in

* **Check available documentation**
  * For the website see ...
  * For adding a game see ..., ... and ...

If you think that actually reduces readability, then the website "extras" should be separated out the same way that we reference world docs.

Reply:
I think I will separate it on a new sentence, thanks for noting it.

Reply:
Done